### ISSUE TITLE
Invalid kubeconfig causes exit with no output

### ISSUE THREAD

**Issue by intjonathan** [helm/helm#6863]
Output of `helm version`:
(None, that's the error. Issue observed in `v3.0.0-rc.1` and master.)

Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.0", GitCommit:"0ed33881dc4355495f623c6f22e7dd0b7632b7c0", GitTreeState:"clean", BuildDate:"2018-09-28T15:18:13Z", GoVersion:"go1.11", Compiler:"gc", Platform:"darwin/amd64"}
Unable to connect to the server: dial tcp 54.164.44.248:6443: i/o timeout
```

An error in the kubeconfig causes helm invocations to exit with code 1 for all commands, and no output is observed. 

Running with `HELM_DEBUG=true` emits the following:
```
helm.go:71: [debug] invalid configuration: [client-key-data or client-key must be specified for admin to use the clientCert authentication method., certificate-authority-data and certificate-authority are both specified for test-clester. certificate-authority-data will override.]
```
This line is actually helpful, but the error causes helm to exit, so it ought to be emitted at FATAL, not DEBUG. 

Separately, I propose that a broken kubeconfig should not result in a fatal error for most helm subcommands like `--help` and `version`.

**Comment by marckhouzam** [comment]
I've hit this as well after deleting my kind cluster but still pointing to it in kubeconfig

**Comment by hickeyma** [comment]
Fixed in PR #6909 merge.
